sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -243,81 +256,52 @@ public ChatThreadAsyncClient getChatThreadClient(String chatThreadId) {
     }
 
     /**
-     * Receive real-time messages and notifications.
+     * Receive real-time notifications.
+     * @param skypeUserToken the skype user token
+     * @param context the app context of the app
```

Comment:
What about using something like this? It's a bit more descriptive and avoids using the word 'app' twice.
```suggestion
     * @param context the Android app context
```